cycli/main.py

```diff
@@ -117,12 +120,57 @@ def run(self):
                 document = cli.run()
                 query = document.text
 
+                print(query)
+
                 if query in ["quit", "exit"]:
                     raise Exception
 
                 elif query == "help":
                     print(help_text())
 
+                elif query == "refresh":
+                    neo4j.refresh()
+
+                elif query == "schema":
+                    neo4j.print_schema()
+
+                elif query == "schema-indexes":
+                    neo4j.print_indexes()
+
+                elif query == "schema-constraints":
+                    neo4j.print_constraints()
+
+                elif query == "schema-labels":
+                    neo4j.print_labels()
+
+                elif query == "schema-relations":
+                    neo4j.print_relationship_types()
+
+                elif re.match('run-([0-9]+) (.*)', query):
```

Comment:
Perhaps define `m = re.match('run-([0-9]+) (.*)', query)` above so it's not done twice?